src/probnum/diffeq/odesolution.py

```diff
@@ -1,176 +1,70 @@
-"""ODESolution object, returned by `probsolve_ivp`
+"""ODESolution interface.
 
+This object is returned by ODESolver.solve().
+
+Provides dense output (by being callable), is sliceable, and collects the time-grid as well as the discrete-time solution.
 Contains the discrete time and function outputs. Provides dense output
```

Comment:
Some things seem to appear twice in this docstring, maybe remove the second "provides dense output..."